docs/v4.0/layers/README.md

```diff
@@ -2,31 +2,50 @@
 
 The deck.gl layer catalog is organized into a couple of sections.
 
+
 ## Base Layer
 
 All deck.gl layers inherit from the `Layer` base class and
-all its props are available to all layers unless otherwise documented.
+its props are available to all layers unless otherwise documented.
+
 
 ## Core Layers
 
-Core layers are the most stable and supported deck.gl layers. They are
+The Core Layers are a group of geospatial visualization focused layers,
 intended to represent a small set of widely applicable data visualization
 building blocks.
 
+The core layers are the most stable and supported deck.gl layers.
+
+
+### Extrusions and 2.5D support.
+
+Some of the Core layers support extrusions and heights. The layers
+that support extrusions are marked with an "Extrusion" tag. TODO
+
+
 ## 64-bit Layers
 
 Core Layers have 64 bit counterparts that can be used to achieve
-higher precision, particularly under high zoom levels.
+higher precision, particularly under high zoom levels (> 1.000.000x)
+at the cost of sacrificing some performance and memory.
+
+Layers that have a 64 bit counterpart are marked with a 64-bit tag. TODO
+
 
 ## Sample Layers
 
-Deck.gl provides a number of sample layers intended to illustrate
-various ideas and approaches to how layers can be designed.
+Deck.gl provides a number of sample layers in the examples folders
+intended to illustrate various ideas and approaches to how layers
+can be designed. These layers sometimes have documentation in the example
+code, but they are not listed here in the official documentation.
+
+To use one of the sample layers an application simply needs to copy it into
+its own source tree, as these layers are not `export`ed by deck.gl.
 
-**Warning**:
-The sample layers are not as actively supported as core layers,
-and could even have breaking changes between minor releases of deck.gl.
+Remarks:
+* Sample layers are not as actively supported as core layers,
```

Comment:
I believe we should use more positive words here. like "Sample layers are demoed here as examples to show what can be achieved by modifying existing core layers of deck.gl. They are not part of the deck.gl and are intended to be used as the final products instead of building blocks of other layers.